packages/core/src/node/cli.ts

```diff
@@ -18,16 +18,16 @@ export interface CliContribution {
 
     configure(conf: yargs.Argv): void;
 
-    setArguments(args: yargs.Arguments): void;
+    setArguments(args: yargs.Arguments): Promise<void>;
```

Comment:
How about `MaybePromise` to avoid breaking existing clients?

Reply:
I didn't know about that, I'll try.